### PR TITLE
TCC-12804: Update IBAN's and Reference Postman collections

### PR DESCRIPTION
**Pull request by rtriska**
## Brief description of what is being done
Updated IBAN's and Reference collection to be consistent with our RAML documentation.

## What areas of the platform are affected by this change
Postman collection

## ToDos
N/A

## Deploy Notes
N/A

### CODE REVIEW

File: api_v2.json [hunk 1]
```diff
@@ -2665,6 +2752,110 @@
...
...esponse**\n\nibans: An array of IBAN objects;\n\n  uuid: IBAN unique identifier.\n  \n  iban\\_code: Code of the IBAN.\n  \n  account\\_id: ID of the account that IBAN belongs to.\n  \n  currency: Currency in which the balance is held, this must be 3 characters long. Currency codes are ISO 4217 standard.\n  \n  account\\_holder\\_name: Account holder name.\n  \n  bank\\_institution\\_name: Bank name.\n  \n  bank\\_institution\\_address: Bank address.\n  \n  bank\\_institution\\_country: Bank country.\n  \n  bic\\_swift: Bank identification code.\n  \n  created\\_at: Date and Time of creation.\n  \n  updated\\_at: Date and Time of last update.\n\npaginationpagination: Pagination details relevant to the search query\n\n  total\\_entries:Total entries for this search query\n \n  total\\_pages:Total pages for this search query\n  \n  current\\_page:Current page of the search query\n  \n  per\\_page:Amount of search results to return per page\n  \n  previous\\_page:Previous page for this search query\n \n  next\\_page:Next page for this search query\n  \n  order\t:Order of the search returned for this search query\n  \n  order\\_asc\\_desc:Order of results returned e.g. 'asc' or 'desc'"
```
Comment (oladan) [inline api_v2.json]: paginationpagination looks like typo

File: api_v2.json [hunk 2]
```diff
@@ -2665,6 +2752,110 @@
...
```
Comment (oladan) [inline api_v2.json]: paginationpagination

File: api_v2.json [hunk 3]
```diff
@@ -2665,6 +2752,110 @@
...
```
Comment (oladan) [inline api_v2.json]: paginationpagination

**Review by oladan — CHANGES_REQUESTED** [review]
paginationpagination looks like typo.
@rtriska  if it's ok - then ping me I'll approve this pr.

**Review by oladan — APPROVED** [review]

**Comment by rtriska** [comment]
@oladan good catch! Fixed all typos, please review once more ♻️

**Pull request merged**